services/common/src/components/project/ProjectDescriptionTab.tsx

```diff
@@ -0,0 +1,472 @@
+            : notApplicableText;
+        const projectSummaryAuthorizationGuid = authorization?.project_summary_authorization_guid;
+
+        return {
+          project_type: projectType,
+          permit_no: permitNo,
+          date_submitted: dateSubmitted,
+          project_summary_authorization_guid: projectSummaryAuthorizationGuid,
+        };
+      });
+
+    setMinesActData(result);
+  };
+
+  const processOtherActAuthorization = (
+    authorization,
+    permitAuthorizationType,
+    projectSummaryAuthorizationType
+  ) => {
+    if (
+      (authorization.ams_status_code === "400" || authorization.ams_status_code === "500") &&
+      authorization.project_summary_authorization_type === projectSummaryAuthorizationType
+    ) {
+      const dateSubmitted = formatDateTimeTz(authorization.ams_submission_timestamp);
+      const permitTypeLabel = parseProjectTypeLabel(authorization.project_summary_permit_type[0]);
+      const projectType = `${parseTransformedProjectSummaryAuthorizationTypes(
+        permitAuthorizationType,
+        projectSummaryAuthorizationType
+      )} - ${permitTypeLabel}`;
+      const permitNo =
+        authorization.project_summary_permit_type[0] === "AMENDMENT"
+          ? authorization.existing_permits_authorizations[0]
+          : notApplicableText;
+      const projectSummaryAuthorizationGuid = authorization?.project_summary_authorization_guid;
+
+      let status = createStatusBadge("Rejected", "error");
+      if (authorization.ams_status_code === "500") {
+        status = createStatusBadge("Failed", "error");
+        setShouldDisplayRetryButton(true);
+      }
+
+      return {
+        project_type: projectType,
+        permit_no: permitNo,
+        ams_tracking_number: notApplicableText,
+        date_submitted: dateSubmitted,
+        project_summary_authorization_guid: projectSummaryAuthorizationGuid,
+        status: status,
+      };
+    }
+    return null;
+  };
+
+  const processOtherActAuthorizations = (
+    authorizations,
+    permitAuthorizationType,
+    projectSummaryAuthorizationType
+  ) => {
+    const filteredResults = authorizations
+      .map((authorization) =>
+        processOtherActAuthorization(
+          authorization,
+          permitAuthorizationType,
+          projectSummaryAuthorizationType
+        )
+      )
+      .filter(Boolean);
+    processedOtherActPermitResult.push(...filteredResults);
+  };
+
+  const processAndSetData = (authorizations, types, actType, setData) => {
+    types.forEach((type) => {
+      processOtherActAuthorizations(authorizations, actType, type);
+    });
+    setData([...processedOtherActPermitResult]);
+    processedOtherActPermitResult = [];
+  };
+
+  const loadOtherActPermitData = (authorizations) => {
+    const environmentalManagementActTypes = [
+      "AIR_EMISSIONS_DISCHARGE_PERMIT",
+      "EFFLUENT_DISCHARGE_PERMIT",
+      "REFUSE_DISCHARGE_PERMIT",
+      "MUNICIPAL_WASTEWATER_REGULATION",
+    ];
```

Comment:
Could we put these and the constants below into enums so they can be shared for future use?

Reply:
enum has been created for this as well